app/controllers/response_controller.rb

```diff
@@ -330,6 +331,17 @@ def redirection
     end
   end
 
+
+  def submitted_question
```

Comment:
I suggest you guys rename it to "initiate_chat" and put this method in review_chats_controller.rb if possible.